modules/programs/ssh.nix

```diff
@@ -206,6 +212,42 @@ let
         '';
       };
 
+      remoteForwards = mkOption {
+        type = types.listOf forwardModule;
+        default = [];
+        example = literalExample ''
+          [
+            {
+              bind.port = 8080;
+              host.address = "10.0.0.13";
+              host.port = 80;
+            }
+          ];
+        '';
+        description = ''
+          Specify remote port forwardings. See
+          <citerefentry>
+            <refentrytitle>ssh_config</refentrytitle>
+            <manvolnum>5</manvolnum>
+          </citerefentry> for RemoteForward.
+        '';
+      };
+
+      dynamicForwards = mkOption {
```

Comment:
I'm not sure how to do this, but it might be good to assert that there are no `host.` options set, as there is only one argument to `DynamicForward`.

Reply:
Assertions can be added using the [`assertions`](https://nixos.org/nixos/manual/index.html#sec-assertions-assertions) option. You can find a number of example in Home Manager and NixOS.

Reply:
Thanks, added assertions.